schemas/public/onpremises-kfd-v1alpha2.json

```
@@ -209,11 +213,11 @@
       "properties": {
         "name": {
           "type": "string",
-          "description": "The name of the host"
+          "description": "A name to identify the host (not the hostname or FQDN)."
```

This name is prefixed to the DNS to calculate the full fqdn of the VM, maybe we need to write something about it?

---

We say something about it in the DNSZone field.

For the Load Balanacers, even though the FQDN is calcualted with the name + DNSZone and set as the `kubernetes_hostname` ansible variable in the inventory file (hosts.yaml), it is not used by the haproxy role.

Anyway, I guess we can say that the FQDN will be calculated using this name:


```suggestion
          "description": "A name to identify the host. This value will be concatenated to `.spec.kubernetes.dnsZone` to calculate the FQDN for the host as `<name>.<dnsZone>`."
```

what do you think @nutellinoit?

---

Yep, seems good